NDTensors/src/array/permutedims.jl

```diff
@@ -0,0 +1,14 @@
+# NDTensors.permutedims
+function permutedims(::Type{<:Array}, M, perm)
+  return @strided Mdest = Base.permutedims(M, perm)
+end
+
+# NDTensors.permutedims!
+function permutedims!(::Type{<:Array}, Mdest, ::Type{<:Array}, M, perm)
+  return @strided Mdest .= Base.permutedims(M, perm)
```

Comment:
```suggestion
  @strided Mdest .= Base.permutedims(M, perm)
  return Mdest
```
since there's no guarantee in-place `@strided` operations would return `Mdest`.